_posts/2021-11-24-sprint-report-2.md

```diff
@@ -0,0 +1,62 @@
+---
+author: Tyler Weaver
+comments: false
+date: 2021-11-24 17:00:00+07:00
+layout: post
+slug: sprint-report-2
+title: MoveIt Sprint Report
+description: Nov 5 - Nov 25
+media_type: image
+media_link: /assets/images/blog_posts/2021_11_24_sprint_update.png
```

Comment:
@skolk could you edit this image (or come up with a new one) in some way to make this look better for the website.  Adding some over-laid text to this image might be good enough.